### ISSUE TITLE
Add Feature For Support Protobuf

### ISSUE THREAD

**Issue by hwangjr** [advanced-rest-client/websocket-request#1]
I have used so many test suit for api testing, but still can't find one way for protobuf +socket. so maybe can add this feature?

**Comment by jarrodek** [comment]
It looks like quite complex tasks. 
I will keep this in backlog.

**Comment by hwangjr** [comment]
yes, so if we can figure out a way for this test, this would be awesome.
